spec/react_on_rails/configuration_spec.rb

```diff
@@ -73,23 +73,24 @@ module ReactOnRails
     end
 
     describe ".build_production_command" do
-      it "if configured, ENV[\"WEBPACKER_PRECOMPILE\"] gets set to \"false\"" do
-        expect(ENV["WEBPACKER_PRECOMPILE"]).to be_nil
-
-        ReactOnRails.configure do |config|
-          config.build_production_command = "a string or a module"
-        end
-
-        expect(ENV["WEBPACKER_PRECOMPILE"]).to eq("false")
-        ENV["WEBPACKER_PRECOMPILE"] = nil
+      it "fails when \"webpacker_precompile\" is truly" do
+        allow(Webpacker).to receive_message_chain("config.webpacker_precompile?")
+          .and_return(true)
+        expect do
+          ReactOnRails.configure do |config|
+            config.build_production_command = "a string or a module"
+          end
+        end.to raise_error(ReactOnRails::Error, /webpacker_precompile: false/)
       end
 
-      it "if not configured, ENV[\"WEBPACKER_PRECOMPILE\"] remains nil" do
-        expect(ENV["WEBPACKER_PRECOMPILE"]).to be_nil
-
-        ReactOnRails.configure {} # rubocop:disable-line Lint/EmptyBlock
-
-        expect(ENV["WEBPACKER_PRECOMPILE"]).to be_nil
+      it "doesn't fail when \"webpacker_precompile\" is falsy" do
+        allow(Webpacker).to receive_message_chain("config.webpacker_precompile?")
+          .and_return(false)
+        expect do
+          ReactOnRails.configure do |config|
+            config.build_production_command = "a string or a module"
```

Comment:
1. use a real example command

```
RAILS_ENV=production NODE_ENV=production bin/webpacker
```

2. Provide case of the the build_production_command is blank with both cases of config.webpacker_precompile.

Reply:
done ✅